types/lumino-js-sdk.d.ts

```diff
@@ -1,4 +1,20 @@
 declare module 'lumino-js-sdk' {
+    export interface ChannelIdentifier {
+        tokenAddress: string;
+        partnerAddress: string;
+    }
+    export interface OpenChannelRequest {
+        tokenAddress: string;
+        amountOnWei: number;
+        rskPartnerAddress?: string;
+        rnsPartnerAddress?: string;
+    }
+    export interface BalanceIdentifier {
```

Comment:
I'm not ok with the word identifier here, actually i'm not ok with any of the identifier words on this file, i mean, why identifier? also `ChannelIdentifier` could be confusing since we have something in lumino called ChannelIdentifier already so be careful with the naming here, i think all these structures are requests or parameters.

Reply:
Maybe we could call them just `Payment` and `Deposit` or something without identifier. As makePayments and depositTokens parameters are transient¹ objects just for operations.

I'd like to keep `ChannelIdentifier`, as it's used as id of the channel we want to retrieve

¹ They are actually stored in the db, but from the payer's/depositer's point of view, they act more as parameters of an procedure, than a entity/resource/object to store

Reply:
I understand what you mean, but we already have an identifier for payments actually on some rest calls so i think we shouldn't confuse the user about what is a channel identifier, because if tomorrow we change relationships on db and for some reason we doesn't identify the channels by partner anymore that could lead to confusion, actually this is a filter, you are filtering a channel from the channel list by partner and token, also the result of the rest call includes a channel_identifier field that's a number, that's why i'm just being annoying with this

Reply:
I don't want to speak for @mauricioirace here, but i presume the name `Identifier` is used because it matches the concept; it's just like the columns that make up a PK in SQL tables, in cases where the key is composite.

we also have to be consistent: we can't have ID objects called `Payment` and `Deposit` if we have `ChannelIdentifier` too. and ftr i'd prefer the latter, as `Payment` is too vague and possibly misleading: it's no the actual payment info, just the parameters that identify it.

i'm not particularly bothered about other system using the same names. moreover, why would it be confusing in this case?